templates/ansible/resource.erb

```diff
@@ -187,13 +187,18 @@ def main():
   else
     raise "Ansible does not support create_verb #{object.create_verb}"
   end
+  create_request = if object.custom_create_resource
+                   'resource_to_create'
+                   else
+                     'resource_to_request'
```

Comment:
Does this mean that if custom create and custom update exist, the `resource_to_request` function will still be in the file but not used?

Reply:
Yes it will be.